src/markdownwidget/theme.css

```diff
@@ -0,0 +1,7 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) Jupyter Development Team.
+| Distributed under the terms of the Modified BSD License.
+|----------------------------------------------------------------------------*/
+.jp-EditorWidget {
```

Comment:
This selector will apply the border to all editor widgets, not just the markdown widgets. Is that what you're wanting, or should this be a more specific selector which only targets the markdown widgets?